scripts/eurlex-metadata-scraping.py

```diff
@@ -0,0 +1,31 @@
+import requests
+from lxml import etree
+from mars.db import collections
+import mars.db.db_fields as db_fields
+from mars import logging
+from mars import config
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.getLevelName(config.logging_level))
+
+url = "http://publications.europa.eu/resource/celex/%s?language=%s"
+
+for doc in collections.document_sources.fetchByExample(
+    {db_fields.SOURCE: db_fields.SourceWebsite.eurlex}, batchSize=1000
```

Comment:
Gdy w bazie będzie więcej niż 1000 dokumentów, to ta pętla się wysypie, bo pobranie wszystkich wyników z bazy musi trwać < 30s. Najłatwiej pobrać wszystko za jednym razem. Skoro to tylko url, to raczej ramu nie zabraknie.
```
for doc in list(collections.document_sources.fetchByExample {db_fields.SOURCE: db_fields.SourceWebsite.eurlex}, batchSize=1000)):
```